front-end/src/Components/Reports/reports.tsx

```diff
@@ -0,0 +1,310 @@
+import React, { useMemo } from 'react';
+import {
+  Button,
+  Column,
+  DataTable,
+  DataTableHeader,
+  DataTableRow,
+  DataTableSkeleton,
+  FormLabel,
+  Grid,
+  Pagination,
+  Row,
+  Select,
+  SelectItem,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from 'carbon-components-react';
+import {
+  getBudgets,
+  getCounties,
+  getDepartments,
+  getPrograms,
+  getProjects,
+  getReport,
+  getSites,
+  trackEmployees,
+} from './reports.resource';
+
+export const Report: React.FC = () => {
```

Comment:
Could we name this component better, e.g `EmployeeStatusReport` the component should be more descriptive of the intended work its doing